root/edit/search_macros.tt

```diff
@@ -253,6 +253,35 @@
   </select>
 [% END %]
 
+[% MACRO predicate_vote_count(field, field_contents) WRAPPER wfield predicate="vote_count" %]
+  <select name="args.0" class="arg">
+    [% FOR item=[ [  2,   lp('Approve', 'vote') ]
+                , [  1,   lp('Yes', 'vote') ]
+                , [  0,   lp('No', 'vote') ]
+                , [ -1,   lp('Abstain', 'vote') ]
+                ] %]
+    <option value="[% html_escape(item.0) %]"
+            [% 'selected="selected"' IF field_contents.find_argument(item.0).defined %]
```

Comment:
Cleaned all predicates in https://github.com/metabrainz/musicbrainz-server/pull/497/commits/ec4553cd4de9ef1e7900f0128c1de67ad16c1b2b.